src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -236,6 +236,9 @@
         - name: --enable-network-observability
           type: bool
           short-summary: Enable network observability on a cluster.
+        - name: --enable-advanced-network-observability
+          type: bool
+          short-summary: Enable advanced network observability functionalities on a cluster. Note that enabling this will incur additional costs.
```

Comment:
Actually, you've written a help message for this option in `_params.py` (around line 802). May remove the help there

>         c.argument(
>             "enable_advanced_network_observability",
>             action="store_true",
>             is_preview=True,
>             help="enable advanced network observability functionalities for cluster",
>         )

Reply:
Yes, I meant you've defined the help message of this option `--enable-advanced-network-observability` twice, what you've added in this PR `Enable advanced network observability functionalities on a cluster. Note that enabling this will incur additional costs.` would overwrite what's defined in _params.py (around line 802) `enable advanced network observability functionalities for cluster`.

Reply:
updated